MapboxCoreNavigation/NavigationServiceDelegate.swift

```diff
@@ -146,9 +146,8 @@ public protocol NavigationServiceDelegate: class, UnimplementedLogging {
      
      - parameter service: The navigation service that will alert that user that reducedAccuracy is enabled.
      - parameter manager: The location manager.
-     - returns: A bool indicating whether to alert the user to enable preciseAccuracy.
      */
-    func locationManagerDidChangeAuthorization(_ manager: CLLocationManager)
+    func navigationServiceDidChangeAuthorization(_ service: NavigationService, _ manager: CLLocationManager)
```

Comment:
Only the first argument should ever have a `_` label, and only if the first part of the method name already refers to the argument or makes the method read more like a sentence.